packages/ses/src/module-load.js

```diff
@@ -128,6 +128,14 @@ const loadWithoutErrorAnnotation = async (
 
   const staticModuleRecord = await importHook(moduleSpecifier);
 
+  if (staticModuleRecord === null || typeof staticModuleRecord !== 'object') {
+    assert.fail(
```

Comment:
If this needs to be unusually high speed code, no change suggested.

Otherwise

```suggestion
  assert(staticModuleRecord !== null && typeof staticModuleRecord === 'object',
```
or
```suggestion
 assert.typeof(staticModuleRecord, 'object', ... );
 assert(staticModuleRecord !== null, ...);
```
or something along those lines.

Reply:
I’ve no reason to believe that the difference would make a perceptible impact on performance, but this is within a performance sensitive region of the Agoric runtime that’s struggling to get close to performance parity with nested evaluate bundles, so I think the code errs on the right side of the balance between succinctness and speed.